distribution/docs/src/main/resources/_contents/introductioncontents.adoc

```diff
@@ -0,0 +1,105 @@
+** System Integrators may deploy only those features required.
+* Simplicity of installation and operation.
+** Unzip and run.
+** Configuration via a modern {admin-console}.
+* Simplicity of Development
+** Build simple Plain Old Java Objects (POJOs) and wire them in via a choice of dependency injection frameworks
+** Make use of widely available documentation and components for {branding}'s underlying technologies
+** Modular development supports multi-organizational and multi-regional teams.
+* Federated Open Geospatial Consortium (OGC) Filter-powered Metadata Catalog
+** Enterprise Service Bus (ESB) performs transformation between query standards.
+** Apache Solr integration.
+*** Well Known Text (WKT) indexing and search for spatial awareness
+*** Full text search
+*** XPath searches
+*** XML indexing
+** Open Geospatial Consortium (OGC) KML, CSW, and WFS federated services
+** OpenSearch federated services.
+** REST API for catalog operations.
+** Integrated content framework to store actual products associated with the indexed metadata.
+** Tika parser for extracting metadata from products.
+** Plugin support for pre and post processing on all operations.
+** Eventing for notifications
+** Metrics
+* Security
+** Web Security Service (WSS) functionality that comes with DDF is integrated throughout the system.
+*** SAML 2.0 Web Browser SSO Profile with included IdP server and client.
+*** SAML Security Token Service (STS) based on WS-Trust.
+*** Automatic protection and Single Sign On (SSO) for web applications without modifying the application itself.
+*** Extensible Java/XACML PDP for authorization decisions.
+*** LDAP integration
+**** Included OSGi enabled OpenDJ LDAP server.
+*** CAS integration
+*** X.509 authentication
+*** Basic authentication
+*** SAML authentication
+*** Guest login support
+*** Multiple realm login support to mix and match different authentication types and services between endpoints.
+*** WS-Security, WS-SecurityPolicy, WS-Policy, WS-Trust, WS-SecureConversation, WS-Addressing
+** Provides a Security Framework (a set of APIs that define the integration with the DDF framework).
+** Provides Security Service reference implementations for a realistic end-to-end use case.
+** Role and Attribute based access control.
+** Attribute based filtering for searches performed throughout the system.
+** Federated identity through metadata catalog.
+* Search User Interface
+** 3D WebGL map built off of Cesium.
+** 2D map built off of OpenLayers 3.
+** USNG/MGRS grid support.
+** GeoNames geocoder integrated into both maps.
+** CometD integration for push notifications.
+** Upload and edit capability.
+** Saved workspaces (searches and metadata artifacts).
+* Admin Web User Interface
+** Web based install wizard.
+** Application grid to organize configurations.
+** Pluggable configuration pages for applications to simplify configurations for complex scenarios.
+** Metrics web application to view up to date system metrics.
```

Comment:
This list actually comes from our readme.

https://github.com/codice/ddf#features

That list is long for search engine relevance which is not as important in the documentation.